app/actions/index.js

```diff
@@ -94,6 +104,39 @@ const extractSurveyBundle = (id, bundle, _callback) => {
   bundle.open();
 };
 
+const syncSurveyData = (id, target, dispatch) => {
+  console.log("does this happen", target);
+
+  dispatch({
+    type: types.SYNCING_SURVEY_DATA,
+    id
+  });
+
+  const progressFn = i => {
+    dispatch({
+      type: types.SYNCING_SURVEY_DATA_PROGRESS,
+      progress: i,
+      id
+    });
+  };
+
+  osm.replicate(target, { progressFn }, function(err) {
```

Comment:
If we're using arrow functions generally, maybe switch this to an arrow function?